sdcm/nemesis.py

```diff
@@ -1732,10 +1732,13 @@ def repair_nodetool_rebuild(self):
     def disrupt_nodetool_cleanup(self):
         # This fix important when just user profile is run in the test and "keyspace1" doesn't exist.
         test_keyspaces = self.cluster.get_test_keyspaces()
-        for node in self.cluster.nodes:
+
+        def clean_task(node):
             InfoEvent('NodetoolCleanupMonkey %s' % node).publish()
             for keyspace in test_keyspaces:
                 node.run_nodetool(sub_cmd="cleanup", args=keyspace)
+        with ThreadPoolExecutor() as executor:
```

Comment:
How many executers we are using here ?

Reply:
According to the source code it is as of following: `max_workers = min(32, (os.cpu_count() or 1) + 4)`
Or from `5` to `32` if not specified.